src/app/components/pages/TeacherRegistration.tsx

```diff
@@ -21,11 +21,39 @@ import useRegistration from "../hooks/useRegistration";
 import { RegistrationSubmit } from "../elements/inputs/RegistrationSubmit";
 import ReCAPTCHA from "react-google-recaptcha";
 import { Recaptcha } from "../elements/inputs/Recaptcha";
+import { Link } from "react-router-dom";
 
 const metaDescriptionCS =
   "Sign up for a free account and get powerful GCSE and A Level Computer Science resources and questions. For classwork, homework, and revision.";
 
-export const TeacherRegistrationTerms = ({ acceptConditions }: { acceptConditions: (arg0: boolean) => void }) => {
+const AreYouATeacher = ({ onTeacherConfirmed }: { onTeacherConfirmed: () => void }) => {
+  return (
+    <Container id="confirm-teacher" className="mb-5">
+      <TitleAndBreadcrumb
+        currentPageTitle="Are you a Teacher?"
+        breadcrumbTitleOverride="Teacher"
+        intermediateCrumbs={[REGISTER_CRUMB]}
+        className="mb-4"
+      />
+      <Col>
+        <p className="mt-5 mx-auto col-lg-10">
+          You will be required to provide evidence of being a teacher before your teacher account will be activated. If
+          you are not listed on your school&apos;s website we will be required to contact your school directly.
+        </p>
+        <div className="text-center mb-1 mt-4">
+          <Button onClick={() => onTeacherConfirmed()} className="m-2 m-lg-4 btn btn-success border-0">
+            Yes I am a teacher
```

Comment:
Small one, but for clarity it would be good to add a comma after the "Yes". So: "Yes, I am a teacher". The same for the student: "No, I am a student".